src/interface/web/app/page.tsx

```diff
@@ -83,56 +84,22 @@ function convertColorToTextClass(color: string) {
 }
 
 function convertSuggestionColorToTextClass(color: string) {
-    if (color === 'blue') return `bg-gradient-to-b from-white 50% to-sky-50`;
-    if (color === 'yellow') return `bg-gradient-to-b from-white 50% to-yellow-50`;
-    if (color === 'green') return `bg-gradient-to-b from-white 50% to-green-50`;
-    if (color === 'pink') return `bg-gradient-to-b from-white 50% to-pink-50`;
-    if (color === 'purple') return `bg-gradient-to-b from-white 50% to-purple-50`;
-    return `bg-gradient-to-b from-white 50% to-orange-50`;
-}
-
-function convertSuggestionColorToIconClass(color: string) {
-    if (color === 'blue') return iconMap.Robot('blue', 'w-8', 'h-8');
-    if (color === 'yellow') return iconMap.Globe('yellow', 'w-8', 'h-8');
-    if (color === 'green') return iconMap.Palette('green', 'w-8', 'h-8');
-    else return iconMap.Lightbulb('orange', 'w-8', 'h-8');
+    console.log(color);
+    const colors = ['blue', 'yellow', 'green', 'pink', 'purple'];
+    if (colors.includes(color)) {
+        return ""+`bg-gradient-to-b from-[hsl(var(--background))] to-${color}-100/${color=="green" ? "90" : "70"} dark:from-[hsl(var(--background))] dark:to-${color}-950/30 dark:border dark:border-neutral-700`;
```

Comment:
This won't work because of the way tailwind interprets classnames. Update it to return static class names?

Reply:
So it actually can work! I added a set of allowed patterns to the ```tailwind.config.ts``` file that specifically allows dynamically generating these type of things. Take a look and let me know if you still want to do static classes :)

Reply:
Ahh, interesting. We can leave as is then, but generally I think it would be better to do less custom configuration. Still, this is good to know! Good find.